egs/wsj/s5/steps/decode_tensorflow.sh

```diff
@@ -0,0 +1,82 @@
+# This script does decoding with a transition model
+
+# Begin configuration section.
+stage=1
+nj=4 # number of decoding jobs.  If --transform-dir set, must match that number!
+acwt=0.1  # Just a default value, used for adaptation and beam-pruning..
+cmd=run.pl
+beam=15.0
+max_active=7000
+min_active=200
+lattice_beam=8.0 # Beam we use in lattice generation.
+iter=final
+num_threads=1 # if >1, will use gmm-latgen-faster-parallel
+scoring_opts=
+skip_scoring=false
+minimize=false
+# End configuration section.
+
+echo "$0 $@"  # Print the command line for logging
+
+[ -f ./path.sh ] && . ./path.sh; # source the path.
+. parse_options.sh || exit 1;
+
+if [ $# -ne 3 ]; then
+  echo "Usage: $0 [options] <graph-dir> <data-dir> <decode-dir>"
```

Comment:
this `<data-dir>` doesn't seem to be anything like the normal data-dir... seems to contain loglikes.  You haven't reworked the usage message of this script to explain what it is or to rename it.